pyamplipi/models.py

```diff
@@ -117,6 +117,14 @@ class Stream(BaseModel):
     freq: Optional[str] = None
     client_id: Optional[str] = None
     token: Optional[str] = None
+    server: Optional[str] = None
+    index: Optional[int] = None
+    disabled: Optional[bool] = False
+    ap2: Optional[bool] = False
```

Comment:
I would argue that ap2 should be set to None just to avoid inserting this field in post parsing and confusingly adding a field on a stream that is not airplay. This is probably an error in our spec.